authorization.go

```diff
@@ -0,0 +1,17 @@
+package turnpike
+
+type Authorizator interface {
```

Comment:
LOL.  Already discussed in chat but the filename should also change to match (Go idiom).